src/utilities/highlight/highlight.story.js

```diff
@@ -0,0 +1,39 @@
+import withCode from '@ecl/storybook-addon-code';
+import getSystem from '@ecl/builder/utils/getSystem';
+import { getColorModeControls } from '@ecl/story-utils';
+
+const system = getSystem();
+
+const getArgs = () => {
+  const args = {};
+
+  if (system === 'ec') {
+    args.color_mode = 'default';
+  }
+
+  return args;
+};
+
+const getArgTypes = () => ({
+  ...getColorModeControls(),
+});
+
+export default {
+  title: 'Utilities/Highlight',
+  decorators: [withCode],
+  parameters: {
+    EclNotes: { disable: true },
+  },
+};
+
+export const Highlight = (args) => {
+  let classes = 'ecl-u-highlight';
+  if (system === 'ec') {
+    classes += ` ecl-color-mode--${args.color_mode}`;
```

Comment:
doing this adds by default a ecl-color-mode-- class, i think yo could simply check for the color mode arg itself, being that only in EC without even needing to detect the system here.
But to me, we shouldn't add the color mode class to the element itself, that is not really the right way to use them, it makes you think that the color mode is a sort of a duplicate of the background utilities, it should be set on a parent element i think, to mean that if a color mode is being used then this background is going to react to it.